admin/acceptance_tests/conftest.py

```diff
@@ -21,15 +21,16 @@ def dbsession(settings):
     init_db(engine, force=True)
     session_factory = get_session_factory(engine)
     session = get_tm_session(session_factory, transaction.manager)
-    return session
+    yield session
```

Comment:
Just for info, does it change something?

Reply:
No, but I've used a profiler here, and I've live it like this

Reply:
OK, thanks :-)